pkgs/development/libraries/libtelnet/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "libtelnet-${version}";
+  version = "0.21";
+
+  src = fetchFromGitHub {
+    owner = "seanmiddleditch";
+    repo = "libtelnet";
+    rev = "45f2d5cfcf383312280e61c85b107285fed260cf";
```

Comment:
Please use tags here instead of commit hashes. In this case, you can just use `rev = version;`.